docs/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>todo list</title>
+  <script defer src="main.js"></script></head>
+ 
+<body>
+  <section class="todo-container">
+    <div class="todo-list todo-intro"><h4 class="todo-intro-title">Demo</h4><i class="fa-solid fa-rotate"></i></div>
+    <div class="todo-list todo-add">
+      <input type="text" placeholder="Add to your list">
+      <i class="fa-solid fa-arrow-left-long"></i>
+    </div>
+    <ul class="todo-group"></ul>
+
+      <button type="button" class="clear-todo"><span>Clear all completed</span></button>
```

Comment:
As recommended by HTML best practice, it would be better if there are no empty lines without any reason

Reply:
Thank you for letting me know the changes I need to make to improve my project 👍